### DISCUSSION TITLE
Always returns `null` function `isAuthenticated`

### DISCUSSION THREAD

**Discussion by jcjp** [sergiodxa/remix-auth#277] [Q&A]
I tried following the guide for `FormStrategy` I have a real temporary API that just returns a token to see it the library would work. However upon further testing  `authenticator.isAuthenticated` is always returning `null` even though I explicitly tell it to return everything from the API, I can also console log the return object.

Also when I use the redirect options of the library it goes into an infinite loop of redirection. I have a loader that verifies if the `isAuthenticated` returns the item redirect it to a protected route, and the protected layout has a loader similar but redirects you to the unprotected page.

I have some sample code here: https://github.com/sergiodxa/remix-auth/discussions/276 I closed the discussion thinking I solved the issue but apparently I was able to solve it without using the help of the library by access the data needed on the request itself.

**Comment by sergiodxa** [comment]
Answered in #276
